beater/config/sampling.go

```diff
@@ -49,6 +50,8 @@ type TailSamplingConfig struct {
 	IngestRateDecayFactor float64               `config:"ingest_rate_decay" validate:"min=0, max=1"`
 	StorageGCInterval     time.Duration         `config:"storage_gc_interval" validate:"min=1s"`
 	TTL                   time.Duration         `config:"ttl" validate:"min=1s"`
+	StorageLimit          string                `config:"storage_limit"`
```

Comment:
It doesn't have to be in this PR, but it might be nice to make a `type Bytes uint64` with its own Unpack method which does the parsing.